packages/cyberstorm/src/components/Layout/PackageSearchLayout/PackageSearchLayout.tsx

```diff
@@ -47,11 +47,17 @@ export class Filters {
   setKeywords: React.Dispatch<React.SetStateAction<string[]>>;
   availableCategories: CategoriesProps;
   setAvailableCategories: React.Dispatch<React.SetStateAction<CategoriesProps>>;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+  pageObjectCount: number;
+  setPageObjectCount: React.Dispatch<React.SetStateAction<number>>;
 
   constructor() {
     [this.keywords, this.setKeywords] = useState<string[]>([]);
     [this.availableCategories, this.setAvailableCategories] =
       useState<CategoriesProps>({});
+    [this.page, this.setPage] = useState<number>(1);
```

Comment:
I don't like this. You're supposed to use hooks in very specific places, and a class isn't one of them. This was mentioned somewhere in my commit message oslt when I refactored this stuff, I think this whole `// TODO: OVERKILL???` part should be refactored. Not that this makes things much worse since we already use hooks in this class, but it's more tech debt anyway.

Reply:
True, I'll try to refactor the provider when I get back home 👍